server/service_test.go

```diff
@@ -200,7 +216,7 @@ func TestGetHeader(t *testing.T) {
 		"0x8a1d7b8dd64e0aafe7ea7b6c95065c9364cf99d38470c12ee807d55f7de1529ad29ce2c422e0b65e3d5a05c02caca249")
 	path := getPath(1, hash, pubkey)
 	require.Equal(t, "/eth/v1/builder/header/1/0xe28385e7bd68df656cd0042b74b69c3104b5356ed1f20eb69f1f925df47a3ab7/0x8a1d7b8dd64e0aafe7ea7b6c95065c9364cf99d38470c12ee807d55f7de1529ad29ce2c422e0b65e3d5a05c02caca249", path)
-
+	
```

Comment:
perhaps set your IDE to automatically trim trailing spaces: https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut